### DISCUSSION TITLE
Azure Kubernetes - Github configured as Provider - returns 404 for /_oauth

### DISCUSSION THREAD

**Discussion by ksankar1993** [thomseddon/traefik-forward-auth#274] [Q&A]
I am trying to configure the **traefik-forward-auth** with the given example of [https://github.com/thomseddon/traefik-forward-auth/tree/master/examples/traefik-v2/kubernetes/advanced-separate-pod](https://github.com/thomseddon/traefik-forward-auth/tree/master/examples/traefik-v2/kubernetes/advanced-separate-pod). I mentioned http://**K8SIP**/_oauth (Overlay mode) as the call back URL in Github as mentioned in the [https://github.com/thomseddon/traefik-forward-auth/wiki/Provider-Setup#redirect-uris](https://github.com/thomseddon/traefik-forward-auth/wiki/Provider-Setup#redirect-uris). I get 404 error for /_oauth URL. I could not find any route defined in traefik ingress. Should I have to write any logic on my own to get the access token? Am I missing anything or misconfigured anything? 

Image of ingress routes
![image](https://user-images.githubusercontent.com/84181630/129733352-91b295f3-8105-4b61-b3eb-48ab42c6f382.png)

Error

![image](https://user-images.githubusercontent.com/84181630/129733617-2189df6c-3208-479a-b431-f766c7af3d39.png)

**Comment by wphampton** [comment]
Did you ever figure this out? I am also having the same behavior.
